communities.json

```diff
@@ -765,6 +765,14 @@
       "inviteCode": "sentry",
       "githubUrl": "https://github.com/getsentry/sentry"
     },
+    {
+      "logo": "php-fig.svg",
+      "title": "PHP-FIG",
+      "quote": "The PHP-FIG Discord server is the informal discussion medium for anything related to PSRs and other PHP topics",
```

Comment:
The `"quote"` is intended to be about your experience with Discord, would you mind writing a short blurb about your experience using Discord?